Products/GenericSetup/OFSP/tests/test_exportimport.py

```diff
@@ -18,8 +18,8 @@
 from Products.GenericSetup.testing import BodyAdapterTestCase
 from Products.GenericSetup.testing import ExportImportZCMLLayer
 
-_FOLDER_BODY = """\
-<?xml version="1.0"?>
+_FOLDER_BODY = b"""\
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
That's the default.  I guess "explicit is better than implicit" blesses the change.

Reply:
On the other hand, we should probably leave it out of at least one of the tests to make sure that we stay backward-compatible with existing files that don't have it. I'll take it out of this one.